src/components/Addtransaction.js

```diff
@@ -0,0 +1,92 @@
+import { Button, Modal, InputGroup, Form, FormControl } from 'react-bootstrap';
+import React from 'react';
+import { useState } from 'react';
+import Senddata from './Senddata';
+
+
+function Addtransaction(props) {
+    const options = [
+        {
+            key: 1,
+            label: "Choose Transaction Type",
+            value: "-",
+        },
+        {
+            key: 2,
+            label: "Debit",
+            value: "debit",
+        },
+        {
+            key: 3,
+            label: "Credit",
+            value: "credit",
+        }];
```

Comment:
Always declare these type of constants outside the component, because if it is inside, it will be redeclared on every rerende.

Reply:
done sir.